Address some comments discussed during meeting (#2)
SpencerDawkins/net-collab-rqmts: draft-kwbdgrr-tsvwg-net-collab-rqmts.md

## Privacy Considerations {#privacy}

Encrypted media payloads along with temporary IPv6 addresses between a server and user (client) provide a measure of privacy for the content and the identity of the user.
It should however be noted that media flows (e.g., encrypted video payloads in SRTP) exhibit a pattern of bursts and intervals that amounts to a signature and is vulnerable to frequency analysis.
To avoid this kind of frequency analysis, media sent by the server would need to be scheduled or multiplexed differently to each user/recipient.
This may be possible in transports like QUIC which allows flexibility in scheduling each stream.
Transports like QUIC also fully encrypt the entire stream and therefore no media headers are observable on path either.
The security aspects of the media payload/ transport are not in the scope of these requirements and is described here only to provide context for metadata privacy.
Privacy considerations for the metadata itself should ensure that no additional information about the content is disclosed to the network, and no information about the user of the content is disclosed to the network or server.

> Some metadata, such as the size of a burst of packets, sequence number, and timestamp, can be readily observed or inferred by entities

Reviewer: ```suggestion
Some metadata (e.g., the size of a burst of packets, sequence number, and timestamp) can be readily observed or inferred by entities
```

> along the network path. However, it's essential to recognize that while sequence numbers and timestamps are typically visible in clear-text protocols like TCP, RTP, or SRTP, they may not be directly observable in encrypted protocols such as QUIC. All metadata sent from the

Reviewer: ```suggestion
along the network path. However, it is essential to recognize that while sequence numbers and timestamps are typically visible in clear-text protocols like TCP, RTP, or SRTP, they may not be directly observable in encrypted protocols such as QUIC. All metadata sent from the
```

To avoid this kind of frequency analysis, media sent by the server would need to be scheduled or multiplexed differently to each user/recipient.
This may be possible in transports like QUIC which allows flexibility in scheduling each stream.
Transports like QUIC also fully encrypt the entire stream and therefore no media headers are observable on path either.
The security aspects of the media payload/ transport are not in the scope of these requirements and is described here only to provide context for metadata privacy.
Privacy considerations for the metadata itself should ensure that no additional information about the content is disclosed to the network, and no information about the user of the content is disclosed to the network or server.

Some metadata (e.g., the size of a burst of packets, sequence number, and timestamp) can be readily observed or inferred by entities
along the network path. However, it is essential to recognize that while sequence numbers and timestamps are typically visible in
the clear-text headers of protocols (e.g., TCP, RTP, or SRTP) they are not directly observable in encrypted protocols such as QUIC. All metadata sent

> from the server to the wireless router, including these elements and others, are vulnerable to modification while in transit. Only an

Reviewer: ```suggestion
from the server to the wireless router, including these elements and others, are vulnerable to modification while in transit. An
```

Without a signaling in place between a receiving host and its network, remote peers are able to mark packets that interfere with the desires of the receiving host -- making their flows more important than what the receiving host considers more important.
This eventually causes all flows to be marked as important, or -- more likely -- such priority markings to be ignored.

> However, prioritizing between flows is presents challenges because the host can have both malicious and legitimate applications, and the remote peers can also be malicious and benign.

Reviewer: ```suggestion
However, prioritizing between flows presents challenges because the host can have both malicious and legitimate applications, and the remote peers can also be malicious and benign.
```
